packages/acis_thermal_check/test_unit.py

```diff
@@ -1,2 +1,2 @@
 import testr
-testr.testr('acis_thermal_check/tests/dpa', 'acis_thermal_check/tests/acisfp')
+testr.testr('-k', 'acisfp or dpa')
```

Comment:
This seems to be enough to get the acisfp and dpa tests (which also have that in the names of the test files).  I don't know if we want to trim down further, but this seems fine for now.

Reply:
Does this mean there would be no way to run other pytests with testr?

Reply:
I think so, given the current infrastructure (i.e. we don't support much command-line configurability). But the idea was that this should be sufficient to verify basic functionality and obvious problems like a packaging issue. ACIS will always be running the full regression suite.

Reply:
I don't understand the question.

Reply:
I ask because if we enable testr to run all tests, then we can change acisops workflows so they are run as part of their own test/release workflow. I am sure they would benefit, but I am not so sure they are aware.

Reply:
Do you mean the issue being that because we limit the tests with this change that they will lose the option to add the tests?  Seems like the easiest thing would just be to add another test*py with the full tests but skip by default?

Reply:
yes, or add multiple test*.py, one for each component.